rust/arrow/src/ipc/convert.rs

```diff
@@ -124,8 +124,11 @@ pub fn fb_to_schema(fb: ipc::Schema) -> Schema {
 
 /// Deserialize an IPC message into a schema
 pub fn schema_from_bytes(bytes: &[u8]) -> Option<Schema> {
```

Comment:
Should we change this function to return `Result<Schema>` instead?

Reply:
DONE